tools/generate-proto.sh

```diff
@@ -2,6 +2,11 @@
 
 set -euo pipefail
 
+if [ -v DEB_HOST_GNU_TYPE ]; then
```

Comment:
I suggest we use the more traditional `[ -n ${DEB_HOST_GNU_TYPE} ]` if you don’t mind, let’s keep our shell code idiot-proof for my future self :)

Reply:
it's technically different as `-v` is true on FOO="", while `-n` is not, but indeed that's not our problem :)

Reply:
yeah, it’s declared version assigned to not empty, but in our case, we want to use the value and it not being empty if we are on that branch, so meh :p